checkipa

```diff
@@ -255,7 +275,10 @@ class IntegrityCheck(object):
                 txt.append('  %s' % (udid))
         else:
             txt.append('  Provisioned Devices (0):')
-
+        txt.append('  Package size: %2.2f GB' % (self.ipa_size/1000000000.0))
```

Comment:
File sizes/prefixes are all powers of 2 not powers of 10. Therefor to convert to GB from what is assumed to be bytes is dividing by 1073741824 or 2^30.

Same goes for the 4GB check later on 4GB is 4294967296 bytes (4*2^30) not 4000000000 bytes. It works in that it wont allow an ipa above the known limit, but is further limiting them by almost a third of a GB.